hr_holidays_legal_leave/tests/test_holidays_legal_leave.py

```diff
@@ -52,3 +52,11 @@ def test_setting_remaining(self):
         # let's attempt setting remaining leave
         self.employee.write({'remaining_leaves': 20})
         self.assertEqual(self.employee.remaining_leaves, 20)
+
+    def test_create_employee(self):
+        '''Check that we are able to create new employee
```

Comment:
docstring should be put with double quote `"` instead of simple according PEP257